client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/usePersonalInfoTab.ts

```diff
@@ -64,14 +64,14 @@ const usePersonalInfoTab = (parameters: usePersoanlInfoTabParameters): usePerson
     }
 
     const getStreetsByCity = (cityId: string) => {
-        axios.get('/addressDetails/city/' + cityId + '/streets').then((res: any) => {
-            setStreets(res && res.data && res.data.map((node: any) => (
-                {
-                    displayName: node.displayName,
-                    id: node.id
-                }
-            )));
-        });
+        // axios.get('/addressDetails/city/' + cityId + '/streets').then((res: any) => {
+        //     setStreets(res && res.data && res.data.map((node: any) => (
+        //         {
+        //             displayName: node.displayName,
+        //             id: node.id
+        //         }
+        //     )));
+        // });
```

Comment:
why 🤷‍♀️

Reply:
Oh I thought I reverted this 😅. I had issues with the request crashing the client. I'll change it back and see whether it keeps happening.

Reply:
I uncommented the code and noticed the error happens when the cityId is an empty string so I added a validity check.